File: Client/Window/angular/projects/main/src/app/components/damage-test-menu/interfaces/damage-test-weapon.ts
import { WeaponHash } from '../../lobbychoice/enums/weapon-hash.enum';

export interface DamageTestWeapon {
    /** Weapon */
    [0]: WeaponHash;

    /** Damage */
    [1]: number;

    /** HeadshotMultiplicator */
    [2]: number;
}

File: Server/Data/Models/DamageTestWeapon.cs
using GTANetworkAPI;
using Newtonsoft.Json;

namespace TDS_Server.Data.Models
{
    public class DamageTestWeapon
    {
        [JsonProperty("0")]
        public WeaponHash Weapon { get; set; }

        [JsonProperty("1")]
        public float Damage { get; set; }

        [JsonProperty("2")]
        public float HeadshotDamageMultiplier { get; set; }
    }
}
